### ISSUE TITLE
COAM\Payment History - Internal User can view payment history

### ISSUE THREAD

**Issue by shon-button** [bcgov/cas-coam#46]
#### Description:

As an internal user, I can see the payment history for an operation’s compliance obligation 

#### Acceptance Criteria:

See [See External User AC](https://github.com/bcgov/cas-coam/issues/20)

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by
- Blocking

**Comment by nanyangpro** [comment]
Link to design is updated in the ticket description

**Comment by BCerki** [comment]
To test this:
- as an industry user, create a report with an obligation
- create a supplementary report with a lower obligation (this is to create an adjustment)
- create a penalty (see generating_penalties.py)
- log in as an IDIR user
- navigate to Compliance > Invoices
- you should see a grid matching the design with all the columns appropriately populated (note that linking this invoice is a separate ticket: https://github.com/bcgov/cas-compliance/issues/195)

E.g.:

<img width="2031" height="1122" alt="Image" src="https://github.com/user-attachments/assets/de688b8d-fcd6-4cf0-a497-640195f2f76a" />

**Comment by fviduya** [comment]
I have tested this in the DEV environment, [here is the test document for reference](https://buttoninc-my.sharepoint.com/:w:/g/personal/francesca_button_is/Ee_W9UHDKv5Csa3lnC8DkfMBf1QjiVdlDeom5pAIpIXRPg?e=0ejQAZ).

I can verify that:
- [x] As internal user, I can view a table showing invoice information to date for all operations on the Invoices tab.
- [x] The design of the invoice grid is in line with the [Figma design](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=25725-4585&t=yDrv1yYygRBJxncc-0)

Noting that this ticket displays only the invoices, linking the invoice is covered under #195 
Moving to Pending PO for review of test. Thank you!

**Comment by vesselak** [comment]
Reviewed testing documentation and accepting ticket.
